### ISSUE TITLE
Silently doesn't remove port context

### ISSUE THREAD

**Issue by vinzent** [voxpupuli/puppet-selinux#164]
## Affected Puppet, Ruby, OS and module versions/distributions

- Module version: v0.7.1

## How to reproduce (e.g Puppet code you use)

```
selinux::port { 'puppet_selinux_test_policy_port_t/tcp':
  context => 'puppet_selinux_test_policy_port_t',
  port => '55555',
  protocol => 'tcp',
  argument => '-d',
}
```

## What are you seeing

```
[root@centos-72-x64 ~]# puppet apply /tmp/test.pp 
...
Notice: Compiled catalog for centos-72-x64.local in environment production in 0.40 seconds
Notice: Applied catalog in 1.03 seconds
``` 

port context is still defined:

```
[root@centos-72-x64 ~]# semanage port --locallist -l
SELinux Port Type              Proto    Port Number

puppet_selinux_test_policy_port_t tcp      55555
```

## What behaviour did you expect instead

port context should be removed

## Any additional information you'd like to impart

* I don't like the `$arguments` param. it should really be something like ensure => (present|absent)
* related to https://github.com/voxpupuli/puppet-selinux/issues/147

**Comment by oranenj** [comment]
Is this still an issue? Wasn't this fixed by the port provider? :)

**Comment by vinzent** [comment]
fixed wiht #174
